test-resources/odata-service-specs/index.ts

```diff
@@ -0,0 +1,3 @@
+/* Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. */
+
+export const oDataServiceSpecs = __dirname;
```

Comment:
could you please share the use case of this?

Reply:
I used it in the getTestService to read a service spec. I found many read commands in our tests like: 
```
 edmxPath:
              '../../test-resources/odata-service-specs/v2/API_TEST_SRV/API_TEST_SRV.edmx'
```

I would like to replace them all in a seperate PR because using the variable makes refactoring easier.